query13.sql

```diff
@@ -6,4 +6,7 @@
 */
 
 -- Enter your SQL query here
-select ...
+select *
+from public.a_tgbqkercddrih5lwa1zg
+order by the_geom <-> st_setsrid(st_makepoint(-75.19266128540039, 39.95223725519681), 4326) DESC
```

Comment:
Note that the correct station will _probably_ be listed as relative distances within a small area will likely be maintained using SRID 4326, but the distances themselves will be wrong.